fboss/platform/configs/anacapa/platform_manager.json

```diff
@@ -0,0 +1,1429 @@
+          "slotType": "SMB_R_SLOT",
+          "outgoingI2cBusNames": [
+            "MCB_IOB_I2C_MASTER_9",
+            "MCB_IOB_I2C_MASTER_21",
+            "MCB_IOB_I2C_MASTER_22",
+            "MCB_IOB_I2C_MASTER_24",
+            "MCB_IOB_I2C_MASTER_30"
+          ]
+        }
+      }
+    },
+    "BMC": {
+      "pluggedInSlotType": "RUNBMC_SLOT",
+      "i2cDeviceConfigs": [
+        {
+          "busName": "INCOMING@0",
+          "address": "0x4a",
+          "kernelDeviceName": "lm75b",
+          "pmUnitScopedName": "RUNBMC_THERMAL_SENSOR"
+        }
+      ]
+    },
+    "NETLAKE": {
+      "pluggedInSlotType": "COMESE_SLOT",
+      "i2cDeviceConfigs": [
+        {
+          "busName": "INCOMING@0",
+          "address": "0x11",
+          "kernelDeviceName": "tda38640",
+          "pmUnitScopedName": "COME_HWMON1"
+        },
+        {
+          "busName": "INCOMING@0",
+          "address": "0x22",
+          "kernelDeviceName": "tda38640",
+          "pmUnitScopedName": "COME_HWMON2"
+        },
+        {
+          "busName": "INCOMING@0",
+          "address": "0x45",
+          "kernelDeviceName": "tda38640",
+          "pmUnitScopedName": "COME_HWMON3"
+        },
+        {
+          "busName": "INCOMING@0",
+          "address": "0x66",
+          "kernelDeviceName": "tda38640",
+          "pmUnitScopedName": "COME_HWMON4"
+        },
+        {
+          "busName": "INCOMING@0",
+          "address": "0x76",
+          "kernelDeviceName": "xdpe15284",
+          "pmUnitScopedName": "COME_HWMON5"
+        },
+        {
+          "busName": "INCOMING@1",
+          "address": "0x48",
+          "kernelDeviceName": "tmp75",
+          "pmUnitScopedName": "COME_INLET_TSENSOR"
+        },
+        {
+          "busName": "INCOMING@1",
+          "address": "0x4a",
+          "kernelDeviceName": "tmp75",
+          "pmUnitScopedName": "COME_OUTLET_TSENSOR"
+        }
+      ]
+    },
+    "RTM_L": {
+      "pluggedInSlotType": "RTM_L_SLOT",
+      "i2cDeviceConfigs": [
+        {
+          "busName": "INCOMING@0",
+          "address": "0x33",
+          "kernelDeviceName": "anacapa_rtmcpld",
+          "pmUnitScopedName": "RTM_L_CPLD"
+        },
+        {
+          "busName": "INCOMING@1",
+          "address": "0x71",
+          "kernelDeviceName": "pca9548",
+          "pmUnitScopedName": "RTM_L_MUX_1",
+          "numOutgoingChannels": 8
+        },
+        {
+          "busName": "RTM_L_MUX_1@1",
+          "address": "0x70",
+          "kernelDeviceName": "xdpe15284",
+          "pmUnitScopedName": "RTM_L_ASIC_VOLTAGE_1_1"
+        },
+        {
+          "busName": "RTM_L_MUX_1@3",
+          "address": "0x70",
+          "kernelDeviceName": "xdpe15284",
+          "pmUnitScopedName": "RTM_L_ASIC_VOLTAGE_1_3"
+        },
+        {
+          "busName": "RTM_L_MUX_1@4",
+          "address": "0x70",
+          "kernelDeviceName": "xdpe15284",
+          "pmUnitScopedName": "RTM_L_ASIC_VOLTAGE_1_4"
```

Comment:
Can the names be more descriptive on what these volatages represent? I am not able to tell how RTM_L_ASIC_VOLTAGE_1_1 is different from RTM_L_ASIC_VOLTAGE_1_3, other than they are on different buses